app/uk/gov/hmrc/voapropertylinking/models/modernoised/agentrepresentation/AgentOrganisation.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.voapropertylinking.models.modernoised.agentrepresentation
+
+import play.api.libs.json.{Json, OFormat}
+
+case class AgentOrganisation(
+                         id: Long,
+                         governmentGatewayGroupId: String,
```

Comment:
Is governmentGatewayGroupId required?

Reply:
yes it is part of the object.

Reply:
governmentGatewayGroupId is not required, I will remove it. Good spot